web/docs/connectors/s3.md

```diff
@@ -0,0 +1,45 @@
+# s3
+
+Loads bytes from an Amazon S3 object. Saves bytes to an Amazon S3 object.
+
+## Synopsis
+
+Loader:
+
+```
+s3 [-x|--anonymous] <uri>
+```
+
+Saver:
+
+```
+s3 [-x|--anonymous] <uri>
+```
+
+## Description
+
+The `s3` loader connects to an S3 bucket to acquire raw bytes from an S3
+object. The `s3` saver writes bytes to an S3 object in an S3 bucket.
+
+The connector tries to retrieve the appropriate credentials using AWS's
+[default credentials provider
+chain](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials-chain.html).
+
+### `<uri>` (Loader, Saver)
+
+The path to the S3 object.
+
+The syntax is `s3://<bucket-name>/<full-path-to-object>`.
+
```

Comment:
Addition suggestion:

If you want to use non-AWS endpoints that are S3-compatible, you can pass the alternative endpoint in the URL like that: `s3://<bucket-name>/<full-path-to-object>?endpoint_override=<endpoint_url>`. [More information](https://arrow.apache.org/docs/python/filesystems.html#s3) about URL parameters.

Reply:
I added `?endpoint_override=<endpoint_url>` to the documentation.

Reply:
Actually, Arrow's S3 filesystem has many other such options. I extended the documentation.